vm-mock/vm.js

```diff
@@ -80,28 +80,55 @@ let callStack = callerAddress + ' , ' + contractAddress;
 let ledger;
 let adminContext = false;
 
+let webModule;
+
+const scCallMockStack = [];
+let callCoins = 0n; // Default value, coins for a call
+let chainIdMock = 77658366n; // Default value, chain id for Buildnet
+
 /**
  * Reset the ledger
  */
 function resetLedger() {
   ledger = new Map();
-  ledger.set(callerAddress, {
+  ledger.set(defaultCallerAddress, {
     storage: new Map(),
     contract: '',
-    balance: BigInt(100000),
+    balance: 100_000n,
```

Comment:
i just kept the current implementation. changing this will break a lot of existing tests...
In the end, i think its not a bad thing to have a non zero balance by default. it avoid to mock balance for every unit tests